packages/manager/modules/pci/src/projects/project/storages/snapshots/snapshots.html

```diff
@@ -0,0 +1,81 @@
+<div
+    data-ui-view>
+    <div class="p-5">
```

Comment:
Shouldn't be a `<section>`? Hard to figure out if existing or not some h1 parents:)